gammapy/scripts/fermi_3FGL_info.py

```diff
@@ -0,0 +1,32 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+from __future__ import print_function, division
+import logging
+logging.basicConfig(level=logging.INFO, format='%(levelname)s - %(message)s')
+import os
+from astropy.units import Quantity
+from astropy.io import fits
+from ..utils.scripts import get_parser
+
+__all__ = ['fermi_3FGL_info']
+
+
+def print_info(source_name):
```

Comment:
Sorry for not being clear here.

I meant this: add a new file `gammapy/scripts/catalog_query.py` and add the following line [here](https://github.com/gammapy/gammapy/blob/master/setup.py#L84):
```
    'gammapy-catalog-query = gammapy.scripts.catalog_query:main',
```

Then in `gammapy/scripts/catalog_query.py` implement a command line tool that will for now allow to print and plot info from the 3FGL catalog (can be extended for other catalogs over the coming months), e.g. like this:
```
gammapy-catalog-query --catalog 3FGL --source '3FGL J0349.9-2102' --print-info
gammapy-catalog-query --catalog 3FGL --source '3FGL J0349.9-2102' --plot-lightcurve lightcurve.png
gammapy-catalog-query --catalog 3FGL --source '3FGL J0349.9-2102' --plot-spectrum spectrum.png
```
If you're not interested in writing the command line tool, that's perfectly fine, I can add it later.

Concerning the `Fermi3FGLObject.info` method: I was thinking about printing some nicely formatted info, like in this example (but not all of this is needed, start small):
http://tevcat.uchicago.edu/LATCat2/spectra/lat_0605.html
Of course a two-step approach would be even nicer, where you have one method or property `info_dict` that returns all info in a Python `dict` and then a second one `info` that takes this dict as input and prints it.
This would allow people to both print the info or e.g. store it in JSON with one line of code and send it to a web page for display (something I want to build later).
But it's a bit more work to write down both methods, so up to you.